gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/lms.html

```diff
@@ -81,53 +76,80 @@
         <a href="{% url 'curriculum_list' groupid  %}" tabindex="6"> <i class="fa fa-sitemap ">  </i> {% trans "Topic Map" %}</a>
       </li>
     {% endif %}
+    {% comment %}
+    <li>
+      <a href="{% url 'gtask' group_id  %}"  tabindex="7" > <i class="fa fa-list-alt"></i> {% trans "Task" %}</a>
+    </li>
+
+
+    <li>
+      <a href="{% url 'course_about' group_id %}" {% if title == 'about' %}class="selected"{% endif %} tabindex="1" title="Course Summary"><i class="fi-eye"></i> {% trans "Overview" %} </a>
+    </li>
+    <li>
+      <a href="{% url 'course_content' group_id %}" {% if title == 'course content' or title == 'unit_authoring' %}class="selected"{% endif %} tabindex="2" title="List of all the lessons"><i class="fi-list-thumbnails"></i>
+      {% if not group_object.project_config.tab_name %}{% trans "Lessons" %}{% else %}{{group_object.project_config.tab_name}} {% endif %}</a>
+      {% if is_gstaff %}<a href="{% url 'unit_detail' groupid %}" title="Edit Structure" style="background-color:#6f0859; border-radius:4px; margin-left:-13px; width:0px;"><i class="fa fa-pencil" style="margin-left:-7px; color:white;"></i>{% endif %}</a>
+     </li>
+    {% if 'Group' not in group_object_member_of_names_list  or  'base_unit'  in group_object_member_of_names_list  %}
+    <li>  <a href="{% url 'course_raw_material' group_id %}" {% if title == 'raw material' or title == "raw_material_detail"  %}class="selected"{% endif %} tabindex="3" title="Contains useful files such as images, audios, videos etc."><i class="fi-folder "></i> {% trans "Resources" %}</a> </li>
+    {% endif %}
+    <li>
+    <li>  <a  href="{% url 'course_gallery' group_id %}" {% if title == 'gallery' or title == "asset_gallery_detail" %} class="selected" {% endif %} tabindex="4" title="Upload and share your files such as images, audios, videos, spreadsheets"> <i class="fi-photo "></i>{% if 'Group'  in group_object_member_of_names_list and 'base_unit' not in group_object_member_of_names_list %} {% trans "E-Library" %} {% else %} {% trans "Gallery" %}{% endif %}</a> </li>
+    <li>
+      <a href="{% url 'course_notebook' group_id %}" {% if title == 'notebook' %}class="selected"{% endif %} tabindex="5" title="Create and share notes, read notes shared by all students"><i class="fi-book "></i>
+      {% if 'Group' in group_object_member_of_names_list and  'base_unit' not in group_object_member_of_names_list %}
+        {% trans "Blog" %}
+      {% else %}
+        {% trans "e-Notes" %}
+      {% endif %}
+      </a>
+    </li>
+  </li>
+    {% endcomment %}
```

Comment:
If this code is not needed, remove it.
If needed in near future, add developers note/comments in it.